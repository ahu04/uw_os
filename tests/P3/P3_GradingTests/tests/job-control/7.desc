Multiple bg, fg (2).  Score: 2