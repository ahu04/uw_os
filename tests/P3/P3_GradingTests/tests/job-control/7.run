sleep 10.0
##sleep 0.2
##assertStat S sleep 10.0
\x1A
##sleep 0.2
##assertStat T sleep 10.0
sleep 10.1
##sleep 0.2
##assertStat S sleep 10.1
\x1A
##sleep 0.2
##assertStat T sleep 10.1
sleep 10.2
##sleep 0.2
##assertStat S sleep 10.2
\x1A
##sleep 0.2
##assertStat T sleep 10.2
jobs
fg 3
##sleep 0.2
##assertStat S sleep 10.2
\x03
##sleep 0.2
##assertStat X sleep 10.2
jobs
bg 2
##sleep 0.2
##assertStat S sleep 10.1
fg 2
##sleep 0.2
##assertStat S sleep 10.1
\x03
##sleep 0.2
##assertStat X sleep 10.1
jobs
fg 1
##sleep 0.2
##assertStat S sleep 10.0
\x03
##sleep 0.2
##assertStat X sleep 10.0
exit
