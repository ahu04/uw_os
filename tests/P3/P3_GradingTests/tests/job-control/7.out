${PROMPT}${PROMPT}${PROMPT}${PROMPT}1: sleep 10.0
2: sleep 10.1
3: sleep 10.2
${PROMPT}${PROMPT}1: sleep 10.0
2: sleep 10.1
${PROMPT}${PROMPT}${PROMPT}1: sleep 10.0
${PROMPT}${PROMPT}